docs/updating/nim-migration/online-builder.md

```diff
@@ -0,0 +1,164 @@
+
+		```bash
+		npm run build
+		```
+
+3. There should be three files in the new `build` folder:
+
+			* `ckeditor.d.ts`,
+			* `ckeditor.js`,
+			* `ckeditor.js.map`.
+
+		Now you can start to remove some unused webpack plugins and update the `webpack.config.js` file.
+
+4. Uninstall the following `devDependencies`:
+
+		```bash
+		npm uninstall \
+			@ckeditor/ckeditor5-dev-translations \
+			@ckeditor/ckeditor5-dev-utils \
+			@ckeditor/ckeditor5-theme-lark\
+			css-loader \
+			postcss \
+			postcss-loader \
+			raw-loader \
+			style-loader \
+			terser-webpack-plugin
+		```
+
+5. Install the following packages:
+
+		```bash
+		npm install --save-dev \
+			css-loader \
+			css-minimizer-webpack-plugin \
+			mini-css-extract-plugin \
+			terser-webpack-plugin
+		```
+
+6. Update the `webpack.config.js` file:
+
+		```js
+		'use strict';
+
+		/* eslint-env node */
+
+		const path = require( 'path' );
+		const TerserWebpackPlugin = require( 'terser-webpack-plugin' );
+		const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+		const CssMinimizerPlugin = require( 'css-minimizer-webpack-plugin' );
+
+		module.exports = {
+			devtool: 'source-map',
+			performance: { hints: false },
+
+			entry: path.resolve( __dirname, 'src', 'ckeditor.ts' ),
+
+			output: {
+				// The name under which the editor will be exported.
+				library: 'ClassicEditor',
+
+				path: path.resolve( __dirname, 'build' ),
+				filename: 'ckeditor.js',
+				libraryTarget: 'umd',
+				libraryExport: 'default'
+			},
+
+			optimization: {
+				minimize: true,
+				minimizer: [
+					new CssMinimizerPlugin(),
+					new TerserWebpackPlugin( {
+						terserOptions: {
+							output: {
+								// Preserve CKEditor&nbsp;5 license comments.
+								comments: /^!/
+							}
+						},
+						extractComments: false
+					} )
+				]
+			},
+
+			plugins: [
+				new MiniCssExtractPlugin( {
+					filename: 'ckeditor.css'
+				} ),
+			],
+
+			resolve: {
+				extensions: [ '.ts', '.js', '.json' ]
+			},
+
+			module: {
+				rules: [
+					{
+						test: /\.ts$/,
+						use: 'ts-loader'
+					},
+					{
+						test: /\.css$/i,
+						use: [ MiniCssExtractPlugin.loader, 'css-loader' ]
+					}
+				]
+			}
+		};
+		```
+
+7. Add the following line to the `sample/index.html` file before other CSS files:
+	
+		```html
+		<link rel="stylesheet" type="text/css" href="../build/ckeditor.css">
+		```
+
+8. Delete the old `build` folder and run the following command to create a new build of CKEditor&nbsp;5.
+
+		```bash
+		npm run build
+		```
+
+9. There should be five files in the new `build` folder:
+
+		* `ckeditor.css`,
+		* `ckeditor.css.map`,
+		* `ckeditor.d.ts`,
+		* `ckeditor.js`,
+		* `ckeditor.js.map`.
+
+The new build has two more files, because the CSS is now separated from the JavaScript file, which should improve performance compared to the old approach. When updating your project that uses the `build` folder, remember to import this new CSS file as well.
```

Comment:
```suggestion
The new build has two more files because the CSS is now separated from the JavaScript file, which should improve performance compared to the old approach. When updating your project that uses the `build` folder, remember to import this new CSS file as well.
```